crates/vision/src/lib.rs

```diff
@@ -6,6 +6,7 @@ pub mod image_receiver;
 pub mod image_segmenter;
 pub mod line_detection;
 pub mod perspective_grid_candidates_provider;
+mod ransac;
```

Comment:
Why not...

```suggestion
pub mod ransac;
```

Reply:
Currently not needed, this is private to the cycler. If we make it publicly available, we should consider creating a new standalone crate for ransac.

Reply:
Ok, sounds reasonable. Let's keep it as is for now.